readme.md

```diff
@@ -267,7 +267,7 @@ Thanks for contributing! 🦋🙌
 - [](# "dim-bots") [Dims commits and PRs by bots to reduce noise.](https://user-images.githubusercontent.com/1402241/65263190-44c52b00-db36-11e9-9b33-d275d3c8479d.gif)
 - [](# "esc-to-cancel") [Adds a shortcut to cancel editing a conversation title: <kbd>esc</kbd>.](https://user-images.githubusercontent.com/35100156/98303086-d81d2200-1fbd-11eb-8529-70d48d889bcf.gif)
 - [](# "no-duplicate-list-update-time") [Hides the update time of conversations in lists when it matches the open/closed/merged time.](https://user-images.githubusercontent.com/1402241/111357166-ac3a3900-864e-11eb-884a-d6d6da88f7e2.png)
-- [](# "dim-visited-conversations") [Dims the title of visited conversations, unless you have write access to the repo.](https://user-images.githubusercontent.com/44045911/148573545-39a72dab-a37a-491d-83f3-d6fd015f5dcf.mov)
+- [](# "dim-visited-conversations") [Dims the title of visited conversations.](https://user-images.githubusercontent.com/44045911/149634307-1ed0e554-6957-4afa-8458-9c5bb4f80ebb.mp4)
```

Comment:
The video does not load in Safari for whatever reason. I think it's best to stick with gifs and revert the previous update. I don't want to have to deal with codec issues, gifs just work

<img width="788" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/149751987-877e38e7-f8d7-43cc-8e39-82ab523a2dc3.png">

Reply:
Previous video featured hash changes which is no longer true, so I rerecorded the video.

And I took the chance to convert it into `.mp4` with `ffmpeg` since Chrome doesn't open `.mov` files if opened directly (or in our case, click the link in README). It's fine if they are embedded on GitHub though.

I'm surprised that Safari doesn't support the converted video because macOS does. Maybe there's something wrong with how I convert it, I will take a look later today.

Reply:
It's not too much because, again, I don't want to have debug/review this in the future. Gifs made with licecap are super lightweight too